droidlet/perception/robot/__init__.py

```diff
@@ -6,37 +6,32 @@
     get_color_tag,
 )
 from droidlet.perception.robot.handlers import (
-    InputHandler,
-    MemoryHandler,
-    DetectionHandler,
-    FaceRecognitionHandler,
-    ObjectDeduplicationHandler,
+    ObjectDetection,
+    FaceRecognition,
+    ObjectDeduplicator,
     Detection,
     WorldObject,
     Human,
-    HumanKeypoints,
+    HumanKeypointsOrdering,
+    HumanPose,
 )
 from droidlet.shared_data_structs import RGBDepth
-from droidlet.perception.robot.perception import Perception, SlowPerception
-from droidlet.perception.robot.self_perception import SelfPerception
+from droidlet.perception.robot.perception import Perception
 
 __all__ = [
     Perception,
-    SlowPerception,
-    SelfPerception,
     Detection,
     WorldObject,
     Human,
-    HumanKeypoints,
+    HumanKeypointsOrdering,
     RGBDepth,
     random_colors,
     draw_xyz,
     get_random_color,
     get_coords,
     get_color_tag,
-    InputHandler,
-    DetectionHandler,
-    MemoryHandler,
-    FaceRecognitionHandler,
-    ObjectDeduplicationHandler,
+    ObjectDetection,
```

Comment:
the downside of this naming convention (sans 'handler') is that it is hard to pick out which are structs and which are models (for eg between ObjectDetection vs Detection, HumanPose vs HumanKeypointsOrdering). I prefer more explicit names that convey the intent of the object. One way to fix that would be naming the model-related classes as ObjectDetector, FaceRecognizer and so on. What do you think?

Reply:
We should distinguish them in namespaces rather than via the names of the Structs or Classes themselves. The structs should probably go into a `struct` file then.